src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp

```diff
@@ -26,6 +26,8 @@
 
 #include <map>
 
+using namespace std::string_literals;
```

Comment:
Why?

Reply:
In my opinion
```c++
using namespace std::string_literals;

"Lithuanian eID (PKCS#11)"s
```
clearly expresses the intention: _Create me a std::string from these characters_.

Without `std::string_literals` and `"..."s` you would read

```c++
"Lithuanian eID (PKCS#11)"
```
as _Create me a char array string literal_. _Maybe also convert it implicitly to a `std::string`, but I need to check what the target variable type is to be sure_.

So, it expresses the intent more clearly.